frontend/src/__tests__/cypress/cypress.config.ts

```diff
@@ -52,7 +52,7 @@ export default defineConfig({
     ODH_PRODUCT_NAME: env.ODH_PRODUCT_NAME,
     resolution: 'high',
   },
-  defaultCommandTimeout: 10000,
+  defaultCommandTimeout: 15000,
```

Comment:
lets set this per test that needs it and not globally

Reply:
I meant to remove this change, so good callout. It is removed now as a part of the rebase I've just done.